gitea/attachment.go

```diff
@@ -47,20 +47,14 @@ func (c *Client) GetReleaseAttachment(user, repo string, release int64, id int64
 
 // CreateReleaseAttachment creates an attachment for the given release
 func (c *Client) CreateReleaseAttachment(user, repo string, release int64, file *io.Reader, filename string) (*Attachment, error) {
-	// Read file to upload
-	fileContents, err := ioutil.ReadAll(file)
-	if err != nil {
-		return nil, err
-	}
-
 	// Write file to body
 	body := new(bytes.Buffer)
 	writer := multipart.NewWriter(body)
 	part, err := writer.CreateFormFile("attachment", filename)
 	if err != nil {
 		return nil, err
 	}
-	part.Write(fileContents)
+	io.Copy(part, file)
```

Comment:
The return error should be handled. Sorry, I haven't make it clear on my previous review.